src/libraries/System.Net.Http/src/System/Net/Http/MultipartContent.cs

```diff
@@ -158,6 +158,44 @@ Collections.IEnumerator Collections.IEnumerable.GetEnumerator()
 
         #region Serialization
 
+        // for-each content
+        //   write "--" + boundary
+        //   for-each content header
+        //     write header: header-value
+        //   write content.CopyTo[Async]
+        // write "--" + boundary + "--"
+        // Can't be canceled directly by the user.  If the overall request is canceled
+        // then the stream will be closed an exception thrown.
+        protected override void SerializeToStream(Stream stream, TransportContext? context, CancellationToken cancellationToken)
+        {
+            Debug.Assert(stream != null);
```

Comment:
similar to above, this should throw too. will stop commenting on this, but we should check this for all the `HttpContent` implementations.

Reply:
Fixed, all occurrences.